notebooks/week3/README.md

```
@@ -0,0 +1,70 @@
+- Next is the same as in the previous notebook, we create an endpoint using the model we registred in the same notebook *week 2 - 05.log_and_register_fe_model.p*. This is the model we registred using feature lookup and feature func.
+- When we send request to the model endpoint, this time, we won't need to provide all the features. 3 features will be taken from the feature lookup table, also one feature "house_age" will be calculated by the feature function.
+
+
+### 4. A/B Testing
+```
+04.AB_test_model_serving.py
+```
+In this notebook, we show the setup of A/B testing for two different model versions using Pyfunc. We'll train, register, and implement a serving endpoint that uses a Pyfunc model as a wrapper for these versions.
+
+Steps:
+- We start with loading the configurations, parameters for model A and model B, training and testing datasets.
```

Markdownlint (0.35.0)

Expected: asterisk; Actual: dash
Unordered list style

(MD004, ul-style)